x-pack/plugins/aiops/server/routes/log_rate_analysis.ts

```diff
@@ -474,6 +511,25 @@ export const defineLogRateAnalysisRoute = (
                     abortSignal
                   );
 
+                  if (significantCategories.length > 0) {
+                    const { fields: significantCategoriesFields, df: significantCategoriesDf } =
+                      await fetchTerms2CategoriesCounts(
+                        client,
+                        request.body,
+                        JSON.parse(request.body.searchQuery) as estypes.QueryDslQueryContainer,
+                        significantTerms,
+                        significantCategories,
+                        request.body.deviationMin,
+                        request.body.deviationMax,
+                        logger,
+                        pushError,
+                        abortSignal
+                      );
+
+                    fields.push(...significantCategoriesFields);
```

Comment:
A while ago a [PR](https://github.com/elastic/kibana/pull/157471) was raised about removing spread operations as they are costly.
I would imagine using `concat` will be slightly faster in this situation. 

However, this isn't in a loop, so it probably doesn't make much difference. I had already written the comment, so feel free to ignore :D